lnwallet/rpcwallet/rpcwallet.go

```diff
@@ -513,8 +513,8 @@ func (r *RPCKeyRing) SignMessageCompact(keyLoc keychain.KeyLocator,
 //
 // NOTE: This method is part of the keychain.MessageSignerRing interface.
 func (r *RPCKeyRing) SignMessageSchnorr(keyLoc keychain.KeyLocator,
-	msg []byte, doubleHash bool, taprootTweak []byte) (*schnorr.Signature,
-	error) {
+	msg []byte, doubleHash bool, taprootTweak []byte, tag []byte,
+) (*schnorr.Signature, error) {
```

Comment:
I think it would make sense to swap the commits

Reply:
good call

Reply:
Hmmm so I think this makes sense to undo having to add a dummy []byte value in `server_signer` while the signer request doesn't yet have tag added. But to ensure the new integration test works properly, the itest would then need to be split into a third commit. Sound ok?

Reply:
or squashed moved to the second commit i suppose 🤔

Reply:
yeah either option sounds good :)